vulnfeeds/cpp/main.go

```diff
@@ -0,0 +1,238 @@
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"encoding/json"
+	"flag"
+	"io/ioutil"
+	"log"
+	"net/url"
+	"os"
+	"path"
+	"regexp"
+	"strings"
+	"text/template"
+
+	"cloud.google.com/go/logging"
+	"github.com/google/osv/vulnfeeds/cves"
+	"github.com/google/osv/vulnfeeds/utility"
+)
+
+const projectId = "oss-vdb"
+
+var Logger utility.LoggerWrapper
+
+// Checks if a URL is to a supported repo.
+func IsRepoUrl(url string) bool {
+	re := regexp.MustCompile(`http[s]?:\/\/(?:c?git(?:hub|lab)?)\.|\.git$`)
+
+	return re.MatchString(url)
+}
+
+// Checks if a URL relates to the FSF.
+func IsGnuUrl(url string) bool {
```

Comment:
s/Gnu/GNU/ per https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Same for URL

Reply:
Done